lblod/ember-rdfa-editor-lblod-plugins
# GN-4841 Add ability to specify area locations in the location-plugin

### Overview
Required some work to shield ember-leaflet from being confused by args and some type gymnastics. I'm pretty happy with the UX for something simple, but maybe it's not intuitive enough? I'll try to get leaflet-draw to work.

##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4841

### Setup
N/A

### How to test/reproduce
On the besluit sample, 'insert location', then select area. I'm not going to say how to use it, as it'll help you see if it's intuitive enough. Once you've figured out how to do it, try making different location types, hoping between editing different ones, refreshing, etc.

### Challenges/uncertainties
Still not sure what we want to do on the template builder side as we can't set a label yet, so it's still using the old address plugin.

### Checks PR readiness
